image/gimp_brush.ksy

```diff
@@ -0,0 +1,38 @@
+meta:
+  id: gimp_brush
+  title: GIMP (GNU Image Manipulation Program) brush version 2 file
+  file-extension: gbr
+  license: CC0-1.0
+  endian: be
+doc-ref: https://gitlab.gnome.org/GNOME/gimp/-/raw/master/devel-docs/gbr.txt
+seq:
+  - id: header_size
```

Comment:
```
../formats/gimp_brush.ksy: /seq/0/id: warning: use `len_header` instead of `header_size`, given that it's only used as a byte size of `header` (see https://doc.kaitai.io/ksy_style_guide.html#attr-id)
```

Reply:
ok, fixed